lib/internal/bootstrap/loaders.js

```diff
@@ -285,7 +285,18 @@ NativeModule.prototype.proxifyExports = function() {
     }
   };
 
-  this.exports = new Proxy(this.exports, handler);
+  if (this.exports === process) {
```

Comment:
Can't we just use `if (this.id === 'process')`?

Reply:
I think object identity is more robust than a string identity here.